mail_reply_to_sender/data/server_object_lines.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <data noupdate="1">
+        <record id="create_server_object_line" model="ir.server.object.lines">
+            <field name="col1" ref="mail.field_mail_mail__reply_to"/>
+            <field name="evaluation_type">equation</field>
+            <field name="value">record.email_from</field>
+        </record>
+   </data>
+</odoo>
```

Comment:
base.automation + automation line should be in the same file

Reply:
Done.